ddtrace/profiling/collector/_memalloc.c

```diff
@@ -394,20 +394,18 @@ iterevents_new(PyTypeObject* type, PyObject* Py_UNUSED(args), PyObject* Py_UNUSE
     }
 
     IterEventsState* iestate = (IterEventsState*)type->tp_alloc(type, 0);
-    if (!iestate)
+    if (!iestate) {
+        PyErr_SetString(PyExc_RuntimeError, "failed to allocate IterEventsState");
```

Comment:
Is the exception handled somewhere?

Reply:
Yeah, it's handled here: https://github.com/DataDog/dd-trace-py/blob/b90fa382764908d0bf11c84cbd097ad9f1a49a27/ddtrace/profiling/collector/memalloc.py#L170-L179

Reply:
https://github.com/DataDog/dd-trace-py/blob/71e3997e0eaf549e4c96f2af8b9ceebe01faf3b7/ddtrace/profiling/collector/memalloc.py#L174-L179

Yes, we do handle RuntimeError here.